### DISCUSSION TITLE
Database connection selector

### DISCUSSION THREAD

**Discussion by CLCWebsiteServices** [stefanzweifel/laravel-backup-restore#33] [General]
Hi, 

After seeing your package, of what seems to be very well done! 

How long do you think before you'll be releasing a finished version for the connecting selector inside the `MySql.php` etc.. for the importer?
```php
// @todo: Use $pendingRestore->connection
// $importToDatabase = $pendingRestore->database;
```
It's just I'd much rather stick to your package, rather than creating an abstract & extending it myself in my project for you then to do your own version! 

This comes into discussion because my platform uses Multi-tenancy, and the connectionName is always `tenant`, so would need to extend your MySql, replace the function with a bind and basically write your code again!

**Comment by stefanzweifel** [comment]
@CLCWebsiteServices Thanks for the reminder. I still have a task open in my personal todo-list app to tackle all the open _TODO_ -items in the next couple of days or weeks.

Currently have a free evening, might give this a try.

Looks like I have to adjust the `DbImporter`-class so it accepts the `connection`-name. 🤔

**Reply by CLCWebsiteServices** [reply]
Hi,

That's a damn quick response! and yeah, it seems exactly like that: pass to both that class, then reference it when you call the abstracted class

**Reply by stefanzweifel** [reply]
This _issue_ has been fixed in https://github.com/stefanzweifel/laravel-backup-restore/pull/34 and a new release [`v1.1.1`](https://github.com/stefanzweifel/laravel-backup-restore/releases/tag/v1.1.1) has been released.

**Reply by CLCWebsiteServices** [reply]
Perfecto, I'll review on my end shortly & let you know how it works!
